backend/src/test/java/com/javahelp/backend/data/DynamoDBUserStoreTest.java

```diff
@@ -34,6 +38,8 @@ public void testCreateRead() {
                 "289375034875093",
                 "Erin",
                 "McDonald");
+        clientInfo.setAttribute("Age", "35");
+        clientInfo.setAttribute("Specialty", "Pain");
         User u = new User("test", clientInfo, "test_user");
```

Comment:
This `StringID` should be randomly generated by the User database, but this is fine with me.

Reply:
Correct me if I'm wrong, but I believe it is indeed generated by the database in this case.